tests/Basic/IFrameTest.php

```diff
@@ -6,22 +6,40 @@
 
 final class IFrameTest extends TestCase
 {
-    public function testIFrame(): void
+    /**
+     * @param string $iframeIdentifier
+     * @param string $elementSelector
+     * @param string $elementContent
+     *
+     * @dataProvider iFrameDataProvider
+     */
+    public function testIFrame($iframeIdentifier, $elementSelector, $elementContent): void
     {
         $this->getSession()->visit($this->pathTo('/iframe.html'));
         $webAssert = $this->getAssertSession();
 
         $el = $webAssert->elementExists('css', '#text');
         $this->assertSame('Main window div text', $el->getText());
 
-        $this->getSession()->switchToIFrame('subframe');
+        $this->getSession()->switchToIFrame($iframeIdentifier);
 
-        $el = $webAssert->elementExists('css', '#text');
-        $this->assertSame('iFrame div text', $el->getText());
+        $el = $webAssert->elementExists('css', $elementSelector);
+        $this->assertSame($elementContent, $el->getText());
 
         $this->getSession()->switchToIFrame();
 
         $el = $webAssert->elementExists('css', '#text');
         $this->assertSame('Main window div text', $el->getText());
     }
+
+    /**
+     * @return array
+     */
+    public function iFrameDataProvider()
```

Comment:
please make it static as non-static data providers are deprecated in PHPUnit 10 (even if we still use PHPUnit 9 for now)

Reply:
Done.